src/Sylius/Bundle/AdminBundle/Form/Extension/CatalogPromotionActionTypeExtension.php

```diff
@@ -31,6 +32,16 @@ public function __construct(iterable $actionConfigurationTypes, private Environm
             $this->actionConfigurationTypes[$type] = $formType::class;
             $this->actionTypes['sylius.form.catalog_promotion.action.' . $type] = $type;
         }
+
+        trigger_deprecation(
+            'sylius/admin-bundle',
+            '1.14',
+            sprintf(
+                'The "%s" class is deprecated and will be removed in 2.0.
+                )',
+                self::class,
+            ),
+        );
```

Comment:
```suggestion
        trigger_deprecation(
            'sylius/admin-bundle',
            '1.14',
            'The "%s" class is deprecated and will be removed in Sylius 2.0.',
            self::class,
        );
```
Should work just as well